File: app/src/main/java/moran_company/honestgram/fragments/map/MapMvp.java
package moran_company.honestgram.fragments.map;


import java.util.List;

import moran_company.honestgram.base_mvp.BaseMvp;
import moran_company.honestgram.data.Users;

/**
 * Created by roman on 03.11.2017.
 */

public interface MapMvp {

    interface View extends BaseMvp.View {
        void showUsers(List<Users> usersList);
    }

    interface Presenter extends BaseMvp.Presenter {
        void loadUsers();
    }

}

File: app/src/main/java/moran_company/honestgram/fragments/map/MapMvp.kt
package moran_company.honestgram.fragments.map


import moran_company.honestgram.base_mvp.BaseMvp
import moran_company.honestgram.data.ItemUserOrder

/**
 * Created by roman on 03.11.2017.
 */

interface MapMvp {

    interface View : BaseMvp.View {
        fun showObjects(usersList: List<ItemUserOrder>)

        //void showUsers(List<Users> usersList);

        //void showShippedOrders(List<Orders> list);
    }

    interface Presenter : BaseMvp.Presenter {
        fun loadUsers()

        fun loadShippedOrders()

    }

}